test/system/search_test.rb

```diff
@@ -14,11 +14,11 @@ class SearchTest < ApplicationSystemTestCase
   test 'searching using navbar autocomplete' do
     visit '/'
 
-    fill_in("searchform_input", with: "Canon")
+    fill_in("searchform_input", with: "Kite")
 
-    assert_selector ".typeahead li", text: "Canon A1200 IR conversion at PLOTS Barnraising at LUMCON", wait: 10
+    assert_selector ".typeahead li", text: "kites", wait: 10
```

Comment:
`expected to find visible css ".typeahead li" with text "kites" but there were no matches. Also found "View all", which matched the selector but not all filters. `